source/js/content.js

```diff
@@ -28,7 +28,7 @@ $(function () {
     createTaskForTicket({
       key: $("#key-val").text(),
       summary: $("#summary-val").text().replace(/\s+/, ' ').replace(/^\s*(\S.+\S)\s*$/, '$1'),
-      description: $("#description-val").html()
+      description: $("#description-val").text().replace(/[ ]{2,}/g, '').trim()
```

Comment:
@prometheas Do you mind if I update the eslint file to use the latest AirBnB preset and go through and change these little things in a separate PR?